net.kvirc.KVIrc.yaml

```
@@ -0,0 +1,23 @@
+app-id: net.kvirc.KVIrc
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+rename-icon: kvirc
```

```
-- Installing: /app/share/applications/kvirc.desktop
```

so

```suggestion
rename-icon: kvirc
rename-desktop-file: kvirc.desktop
```

Also the AppStream file is missing.

---

have you suggestted for removal in

---

it is not the same. The rename take the **original** file name, not the target one. I didn't go analyze the output. I was just saying that what you wrote meant it already had the name. Sorry if the documentation is unclear.